src/lib/core/data-table/data-table.ts

```diff
@@ -196,6 +215,32 @@ export class CdkTable<T> implements CollectionViewer {
     }
 
     this._isViewInitialized = true;
+
+    this._StickyRows.changes.subscribe((queryList: QueryList<CdkStickyRow>) => {
```

Comment:
What changes do we anticipate here? Do we have a strategy for adapting if the sticky row's content causes it to increase/decrease in size?